libs/chain-signatures/contract/Cargo.toml

```diff
@@ -60,6 +60,7 @@ k256 = { version = "0.13.4", features = [
     "arithmetic",
     "expose-field",
 ] }
+# TODO(#851): Use upstream library
```

Comment:
I can't find the issue #851.

<img width="736" height="433" alt="screenshot" src="https://github.com/user-attachments/assets/1b519f60-509d-4aab-8302-672169ca83b1" />